tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -469,10 +489,15 @@ def test_check_sfputil_reset(duthosts, enum_rand_one_per_hwsku_frontend_hostname
                     "Run command '{}' failed".format(cmd_sfp_presence_per_intf)
 
             parsed_presence = parse_output(sfp_presence["stdout_lines"][2:])
-            assert logical_intf in parsed_presence, \
-                "Interface is not in output of '{}'".format(cmd_sfp_presence_per_intf)
-            assert parsed_presence[logical_intf] == "Present", \
```

Comment:
@bachalla why did you remove this line "assert parsed_presence[logical_intf] == "Present", \
                "Interface presence is not 'Present' for {}".format(logical_intf)"? Please check.